lib/miscParser.ml

```diff
@@ -51,6 +51,8 @@ let location_compare loc1 loc2 = match loc1,loc2 with
 | Location_sreg _, Location_global  _ -> -1
 | Location_global _, Location_sreg _ -> 1
 
+let dump_value : maybev -> string = ParsedConstant.pp_v
```

Comment:
OCaml programmers usually do not add explicit types, unless there is a good reason (_.e.g._ complex types, explicit restriction...)

Reply:
Thanks! Will be removed.